### DISCUSSION TITLE
Discarding menu options on clicking on new extension

### DISCUSSION THREAD

**Discussion by YathindraNikshipth** [eclipse-theia/theia#14369] [General]
![git-issue-menu-options](https://github.com/user-attachments/assets/ae78da7c-194b-4a30-ae09-45bc887f78e6)

how to stop these menu options from displaying besides the extension name.

**Comment by tsmaeder** [comment]
Hi @YathindraNikshipth unfortunately, I don't know what "Karavan" or the "Integrations" view is that you're showing in the screenshots. I assume these are contributed by some VS Code extensions? Which ones? When toolbar actions show up in toolbars when their "when" enablement expression matches the context of the toolbar (see https://code.visualstudio.com/api/references/when-clause-contexts). Without some more information it's impossible to tell whether this is a problem in Theia or the extensions that are contributing these views and actions. Does the same behavior happen in VS Code?
